bundles/best_practices.rst

```diff
@@ -546,6 +546,78 @@ Beware that templates use a simplified version of the logical path shown above.
 For example, an ``index.html.twig`` template located in the ``templates/Default/``
 directory of the AcmeBlogBundle, is referenced as ``@AcmeBlog/Default/index.html.twig``.
 
+Specifics for a UX bundle
+-------------------------
+
+Here are a few tricks to make your bundle install as a UX bundle.
+
+composer.json file
+~~~~~~~~~~~~~~~~~~
+
+Your ``composer.json`` file must have the ``symfony-ux`` keyword:
+
+.. code-block:: json
+
+    {
+        "keywords": ["symfony-ux"]
+    }
+
+Assets location
+~~~~~~~~~~~~~~~
+
+Your assets must be located in one of the following directories, with a ``package.json`` file so Flex can handle it
+during install/update:
+
+* ``/assets``
```

Comment:
perhaps details what are the pros/cons of each folders? or only define one?

reading this, as a dev coming to know best practices, this will leave me with such questions ^^

Reply:
It depends on your bundle structure, whether the bundle uses the "new" structure or not

Reply:
I'm kinda thinking we should just recommend the "best" one which is `/assets`

Reply:
Would adding "(recommended)" after "/assets" be enough?

Reply:
> Would adding "(recommended)" after "/assets" be enough?

after my comment above ``reading this, as a dev coming to know best practices, this will leave me with such questions ^^`

I would say yes it, the best practice is now more explicit